app/components/select-list.js

```diff
@@ -0,0 +1,84 @@
+import Ember from "ember";
+import ValidatableInput from 'ember-cli-html5-validation/mixins/validatable-input';
+
+export default Ember.Component.extend(ValidatableInput, {
+  content: null,
+  selectedValue: null,
+  pkg: null,
+  index: null,
+  flag: true,
+
+  didRender() {
+    this._super(...arguments);
+    if ( this.pkg != null && this.pkg.packageType != null && this.flag === true){
+      if(this.pkg.notes === null || this.pkg.notes.length === 0) {
+        this.pkg.notes = this.pkg.packageType.get('name');
+      }
+      Ember.$("textarea#"+this.index).val(this.pkg.notes);
+      this.flag=false;
+    }
+
+  },
+
+  didUpdate() {
+    this._super(...arguments);
+    if ( this.flag === false && this.pkg != null && this.pkg.packageType != null && (this.pkg.notes === null || this.pkg.notes.length === 0 ) ){
+      this.flag=true;
+      // this.pkg.notes = this.pkg.packageType.get('name');
+       Ember.$("textarea#"+this.index).val(this.pkg.notes);
+    }
+  },
+
+  // overriden from ember-cli-html5-validation addon
+  inputTagName: function() {
+    return "select";
+  }.property(),
+
+  // overriden from ember-cli-html5-validation addon
+  validate: function() {
+    var input = Ember.$(this.element).find("select")[0],
+      jQueryElement = Ember.$(input);
+
+    if (input.hasAttribute('formnovalidate')) { return; }
+
+    if(input.hasAttribute('required')) {
+      var content = Ember.$.trim(jQueryElement.val());
+
+      if(content.length === 0) {
+        jQueryElement.val('');
+      }
+    }
+
+    if (!input.validity.valid) {
+      this.set('errorMessage', "");
+    } else {
+      this.set('errorMessage', null);
+    }
+
+    input.setCustomValidity('');
+
+    if (!this.get('wasValidated')) {
+      jQueryElement.off('focusout').on('keyup', Ember.run.bind(this, this.validate));
+      this.set('wasValidated', true);
+    }
+  },
+
+  actions: {
+    change() {
+      const changeAction  = this.get('on-change');
```

Comment:
it is a legacy code used by other select-list as you can see in the shared folder.
I can remove it but its consequences might not be in our favour. 
https://github.com/crossroads/shared.goodcity/blob/master/app/components/select-list.js